src/utils/nameUtils.ts

```diff
@@ -125,12 +127,17 @@ function getSuffix(nameType?: NameType) {
  */
 export function normalizeTypeName({ kind, typeName }: TypeDetails) {
   // Only Enum and Composite kinds need normalization
+  let tempTypeName = "";
+  if (typeName.endsWith("[]")) {
+    tempTypeName = "[]"
+    typeName = typeName.replace("[]", "");
+  }
```

Comment:
NIT:

`tempTypeName = "[]"` could be confusing. Could this be rewritten into a boolean variable instead that can be checked in line 136? e.g. `isArray = true;` and then 
```ts
return `${normalizeName(typeName, NameType.Interface)}${isArray ? "[]" : ""}`;
```